src/views/Questions.vue

```diff
@@ -93,7 +106,13 @@ export default {
       });
 
       localStorage.setItem('questions', JSON.stringify(this.questions));
-      // this.tempQuestion
+      this.tempQuestion = '';
+      this.tempAnswer = '';
+    },
+    removeQuestion(quest) {
+      const qIndex = this.questions[this.currentCategory].indexOf(quest);
+      this.questions[this.currentCategory].splice(qIndex, 1);
```

Comment:
I'm not sure if splice works when index is -1. We should better add `if (index !== -1)`